client/src/components/Map.jsx

```diff
@@ -1,9 +1,103 @@
-import { useRef, useEffect } from "react";
+// import { useRef, useEffect } from "react";
+// import { Loader } from "@googlemaps/js-api-loader";
+// import { Box } from "@mui/material";
+
+// const Map = () => {
+//   const mapContainerRef = useRef(null);
+
+//   useEffect(() => {
+//     const loader = new Loader({
+//       apiKey: "AIzaSyCODlhg_73yAwYNpOcjCE3nQOWWy-xioJs",
+//       version: "weekly",
+//     });
+
+//     const initialLocation = {
+//       lat: 51.52205390886717,
+//       lng: -0.08467381645964438,
+//     };
```

Comment:
Better to delete old code. We can get it back from source control

Reply:
done